src/cryptonote_basic/cryptonote_format_utils.cpp

```diff
@@ -910,28 +910,80 @@ namespace cryptonote
     return res;
   }
   //---------------------------------------------------------------
-  bool is_out_to_acc(const account_keys& acc, const txout_to_key& out_key, const crypto::public_key& tx_pub_key, const std::vector<crypto::public_key>& additional_tx_pub_keys, size_t output_index)
+  bool is_pk_out_key(const crypto::key_derivation & derivation, const crypto::public_key & pk, const txout_to_key& out_key)
```

Comment:
The functions not listed in the header should be put in anonymous namespace or declared as `static`.

Reply:
I prefer static.